### DISCUSSION TITLE
翻译文档 user_guides/useful_tools

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#417] [MMagic]
# 翻译文档 user_guides/useful_tools 

翻译[https://github.com/open-mmlab/mmagic/blob/main/docs/en/user_guides/useful_tools.md](https://github.com/open-mmlab/mmagic/blob/main/docs/en/user_guides/useful_tools.md) 为[https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/useful_tools.md](https://github.com/open-mmlab/mmagic/blob/main/docs/zh_cn/user_guides/useful_tools.md) 

## Task Information

**Repository**：MMagic

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：liweiwp

**Claim Time**：2023年06月25日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by tcexeexe** [comment]
Claim the mission

**Reply by crazysteeaam** [reply]
Apologize for any inconvenience. The task has already been claimed by liweiwp, and unfortunately, it cannot be claimed again.

**Reply by crazysteeaam** [reply]
You can choose another task to work on～

**Reply by tcexeexe** [reply]
ok

**Comment by tcexeexe** [comment]
Hello, The task is to translate Chinese into English or English into Chinese?
